cloudtrail/lib/botocore/data/iam/2010-05-08/examples-1.json

```diff
@@ -0,0 +1,1577 @@
+        "id": "028e91f4-e2a6-4d59-9e3b-4965a3fb19be",
+        "title": "To add a client ID (audience) to an Open-ID Connect (OIDC) provider"
+      }
+    ],
+    "AddRoleToInstanceProfile": [
+      {
+        "input": {
+          "InstanceProfileName": "Webserver",
+          "RoleName": "S3Access"
+        },
+        "comments": {
+          "input": {
+          },
+          "output": {
+          }
+        },
+        "description": "The following command adds the role named S3Access to the instance profile named Webserver:",
+        "id": "c107fac3-edb6-4827-8a71-8863ec91c81f",
+        "title": "To add a role to an instance profile"
+      }
+    ],
+    "AddUserToGroup": [
+      {
+        "input": {
+          "GroupName": "Admins",
+          "UserName": "Bob"
+        },
+        "comments": {
+          "input": {
+          },
+          "output": {
+          }
+        },
+        "description": "The following command adds an IAM user named Bob to the IAM group named Admins:",
+        "id": "619c7e6b-09f8-4036-857b-51a6ea5027ca",
+        "title": "To add a user to an IAM group"
+      }
+    ],
+    "AttachGroupPolicy": [
+      {
+        "input": {
+          "GroupName": "Finance",
+          "PolicyArn": "arn:aws:iam::aws:policy/ReadOnlyAccess"
+        },
+        "comments": {
+          "input": {
+          },
+          "output": {
+          }
+        },
+        "description": "The following command attaches the AWS managed policy named ReadOnlyAccess to the IAM group named Finance.",
+        "id": "87551489-86f0-45db-9889-759936778f2b",
+        "title": "To attach a managed policy to an IAM group"
+      }
+    ],
+    "AttachRolePolicy": [
+      {
+        "input": {
+          "PolicyArn": "arn:aws:iam::aws:policy/ReadOnlyAccess",
+          "RoleName": "ReadOnlyRole"
+        },
+        "comments": {
+          "input": {
+          },
+          "output": {
+          }
+        },
+        "description": "The following command attaches the AWS managed policy named ReadOnlyAccess to the IAM role named ReadOnlyRole.",
+        "id": "3e1b8c7c-99c8-4fc4-a20c-131fe3f22c7e",
+        "title": "To attach a managed policy to an IAM role"
+      }
+    ],
+    "AttachUserPolicy": [
+      {
+        "input": {
+          "PolicyArn": "arn:aws:iam::aws:policy/AdministratorAccess",
+          "UserName": "Alice"
+        },
+        "comments": {
+          "input": {
+          },
+          "output": {
+          }
+        },
+        "description": "The following command attaches the AWS managed policy named AdministratorAccess to the IAM user named Alice.",
+        "id": "1372ebd8-9475-4b1a-a479-23b6fd4b8b3e",
+        "title": "To attach a managed policy to an IAM user"
+      }
+    ],
+    "ChangePassword": [
+      {
+        "input": {
+          "NewPassword": "]35d/{pB9Fo9wJ",
+          "OldPassword": "3s0K_;xh4~8XXI"
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- fb2284cc-91fe-4481-84eb-2503f5fc3da0 -->